lib/libc/riscv/sys/cerror.S

```diff
@@ -34,22 +34,22 @@
 
 #include <machine/asm.h>
 __FBSDID("$FreeBSD$");
+#include "SYS.h"
+
+#define FRAME_SIZE (2 * __SIZEOF_POINTER__)
+#define RA_OFFSET 0
```

Comment:
This is a change of behaviour from before; previously, `ra` was at `sp+8`, as `sp+0` is normally reserved for the frame pointer.

Reply:
Is there a canonical RISCV call frame? I can adjust the code to do that if it helps debuggers.

Reply:
From high address it's varargs spill area, then RA, then S0/FP, then locals. If a frame pointer is being used, it points just after the spill of RA (ie RA is at FP[-PTRSIZE] and preivous FP at FP[-2*PTRSIZE]).